Library/Formula/napi.rb

```diff
@@ -0,0 +1,33 @@
+class Napi < Formula
+  desc "Napi Projekt Client and Subotage subtitle format converter both written in bash"
+  homepage "https://github.com/dagon666/napi"
+  url "https://github.com/dagon666/napi/archive/v1.3.4.tar.gz"
+  sha256 "32b8ee39e00e9ed8cbbdb02bb683ccbfb02bd1f109b1a1e9f97e5badbf12c996"
+
+  depends_on "ffmpeg" => :optional
+  depends_on "media-info" => :optional
+  depends_on "mplayer" => :optional
+
+  patch do
```

Comment:
Please add comments briefly explaining what these patches do, when we can expect to remove them, etc.